tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "compilerOptions": {
-    "target": "ES2020",
+    "target": "ESNext",
```

Comment:
percent complete is using Object.groupBy, which is only available in ESNext

Reply:
`target` tells TS what code to output. When TSC is the only build step, I def keep this at ES2020 or ES2022, as ESNext is a continually moving target that many browsers wont support yet. Since it's also going through rollup though, that my not be necessary.

By default, TS adds libs for whatever your target is, which It think is actually what's changing that's helping you here. Since it's not specified, we have `"lib": ["ESNext"]`, but I'm fairly certain if you put target back to ES2020 and add `"lib": ["ESNEXT"]`, it'll get you what you need too. (You might need `"lib": ["ES2022", "DOM", "DOM.Iterable"]`, if we're messing with dom stuff directly, but I don't think we should be.)